operators/endpointmetrics/manifests/prometheus/crd/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+resources:
+- alertmanager_crd_0_53_1.yaml
+- alertmanagerconfig_crd_0_53_1.yaml
+- podmonitor_crd_0_53_1.yaml
+- probe_crd_0_53_1.yaml
+- prometheus_crd_0_53_1.yaml
+- prometheusrule_crd_0_53_1.yaml
+- servicemonitor_crd_0_53_1.yaml
+- thanosruler_crd_0_53_1.yaml
```

Comment:
`0_53_1` is version, right?

Reply:
yes, that's release version for prometheus operator